README.md

```diff
@@ -27,7 +27,8 @@ setup as environment variables.
 | FILTER_API_URL        | http://localhost:22100               | Filter api URL
 | FILTER_API_AUTH_TOKEN | FD0108EA-825D-411C-9B1D-41EF7727F465 | Secret token to use the Filter api
 | DATASET_API_URL       | http://localhost:22000               | Dataset api URL
-| DATASET_API_AUTH_TOKEN| FD0108EA-825D-411C-9B1D-41EF7727F465 | Secret token to use the Dataset api
+| SERVICE_AUTH_TOKEN    | FD0108EA-825D-411C-9B1D-41EF7727F465 | Secret token to use the Dataset api
```

Comment:
Default value needs updating for `SERVICE_AUTH_TOKEN`

Reply:
Update default value and its comment